fms_core/serializers.py

```diff
@@ -35,10 +35,16 @@ class Meta:
 
 class ContainerExportSerializer(serializers.ModelSerializer):
     location = serializers.SlugRelatedField(slug_field='barcode', read_only=True)
+    container_kind = serializers.CharField(source='kind')
 
     class Meta:
         model = Container
-        fields = ('kind', 'name', 'barcode', 'location', 'coordinates', 'comment')
+        fields = ('name', 'container_kind', 'barcode', 'location', 'coordinates', 'comment')
+
+    def to_representation(self, obj):
+        primitive_repr = super(ContainerExportSerializer, self).to_representation(obj)
+        primitive_repr = {k.replace("_", " "): v for k, v in primitive_repr.items()}
+        return primitive_repr
```

Comment:
I would advise against returning JSON where keys have whitespaces, it is prone to errors. You might use this endpoint for something else in the future and to deal with the whitespaces will be additional work.
It is enough that `get_renderer_context()` removes the ` _ ` in the header. So I suggest just remove `to_representation` methods, then the response at `/api/containers/list_export/` will be:
```
[
    {
        "name": "Freezer001",
        "container_kind": "freezer 3 shelves",
        "barcode": "Freezer001",
        "location": null,
        "coordinates": "",
        "comment": ""
    }
]
```
and the export at `/api/containers/list_export/?format_csv`  will return the csv with column names without underscore.